src/main/java/com/hm/petmaster/listener/PlayerAttackListener.java

```diff
@@ -0,0 +1,23 @@
+package com.hm.petmaster.listener;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.entity.Projectile;
+import org.bukkit.entity.Tameable;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+
+public class PlayerAttackListener implements Listener {
```

Comment:
As for _PlayerInteractListener.java_, you could add a short comment about the purpose of the class and add your name in the author tag!